src/main/java/formflow/library/ScreenController.java

```diff
@@ -91,6 +91,9 @@ ModelAndView getScreen(
     if (currentScreen == null) {
       return new ModelAndView("redirect:/error");
     }
+
+    currentScreen.handleBeforeDisplayAction(submission, uuid);
```

Comment:
I _think_ we need this in the Post as well, maybe after line 160?  Not sure. 

The only reason (that I can think of) that we'd need this is if we display data from previous pages, we will want to make sure they are presentable.  Thoughts?